content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -264,6 +264,102 @@ $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata
 $ kubectl exec -it $SOURCE_POD -c sleep curl http://httpbin.org/headers
 {{< /text >}}
 
+## Install Istio with access to all the external services by default
+
+An alternative to calling external services directly is to instruct Istio proxy to pass through the calls to all the
+external services. This option allows you to start evaluating Istio quickly, without controlling access to external
+services, and decide to [configure access to external services](#configuring-istio-external-services) later.
+
+Istio has an installation option that allows access to all external services for which no HTTP service exist inside
```

Comment:
service exist -> service exists

Reply:
Fixed in https://github.com/istio/istio.io/pull/3140/commits/008967af4acdf44d3470b8679624676294b5fa20